docs/src/configuration/services/elasticsearch.md

```diff
@@ -10,6 +10,15 @@ sidebarTitle: "Elasticsearch"
 
 See the [Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html) for more information.
 
+{{% frameworks %}}
+
+- [Drupal](../../guides/drupal9/elasticsearch.md)
+- [Jakarta EE](../../frameworks/jakarta.md#elasticsearch)
+- [Micronaut](../../guides/micronaut/elasticsearch.md)
+- [Quarkus](../../guides/quarkus/elasticsearch.md)
+- [Spring](../../guides/spring/elasticsearch.md)
+{{% /frameworks %}}
```

Comment:
```suggestion

{{% /frameworks %}}
```

It seems easier to delineate this way. If you like it, I'd apply it to all of them. Not really a breaking issue either way though. :shrug: